src/App.js

```diff
@@ -47,6 +55,13 @@ class App extends Component {
       </div>
     );
   }
+
+  handleAuthenticationInfoUpdate = function(authResponse) {
+    if(authResponse != undefined) {
```

Comment:
Wouldn't this work? `if(!authResponse)`

Reply:
It would. Done.

Reply:
Its not done. I don't see the changes. Did you forget to post the diff?